reliontomo/objects.py

```diff
@@ -373,9 +380,16 @@ def _samplingRateStr(self):
         return "%0.2f Å/px" % self.getCurrentSamplingRate()
 
 
-def createSetOfRelionPSubtomograms(protocolPath, optimSetStar, coordsPointer, template=PSUBTOMOS_SQLITE,
-                                   tsSamplingRate=1, relionBinning=1, boxSize=24, nReParticles=0, are2dStacks=None,
-                                   acquisition=None):
+def createSetOfRelionPSubtomograms(protocolPath: str,
+                                   optimSetStar: str,
+                                   coordsPointer: Pointer,
+                                   template: str = PSUBTOMOS_SQLITE,
+                                   tsSamplingRate: float = 1.0,
+                                   relionBinning: int = 1,
```

Comment:
It's our data model for relionParticles to store the data. It may not mean that all that info is passed to Relion, but this info is needed to properly trace the operations across Reliontomo's workflow.

Reply:
Said that, I may update it to float for robustness. Thanks!